### PR TITLE
Core/PacketIO: Updated logout opcodes

### PR DESCRIPTION
**Pull request by ForesterDev**
(cherry picked from commit f59a701e2903d6ef023cd10aa7f4a80a524a8c07)

**Changes proposed:**

-  Updated logout opcodes to new packet class
- Call SetRoot method instead of direct root/unroot packet sending (as in master branch)

**Target branch(es):** 3.3.5/master

- [x] 3.3.5

**Tests performed:** (Does it build, tested in-game, etc.)
Tested in game, build

### CODE REVIEW

File: src/server/game/Handlers/MiscHandler.cpp [hunk 1]
```diff
@@ -373,18 +372,18 @@ void WorldSession::HandleLogoutRequestOpcode(WorldPacket& /*recvData*/)
     else if (GetPlayer()->duel || GetPlayer()->HasAura(9454)) // is dueling or frozen by GM via freeze command
         reason = 2;                                         // FIXME - Need the correct value
```
Comment (ForesterDev) [inline src/server/game/Handlers/MiscHandler.cpp:373]: Maybe someone knows correct values for reasons? If so I can update this

**Comment by ForesterDev** [comment]
PR is ready. Can be merged

File: src/server/game/Handlers/MiscHandler.cpp (hunk 1 above)
Comment (Faq) [reply src/server/game/Handlers/MiscHandler.cpp:373]: What about this? Anyone?

Comment (ForesterDev) [reply src/server/game/Handlers/MiscHandler.cpp:373]: Apparently no one knows

**Comment by jackpoz** [comment]
Thanks for the PR :)

**Pull request merged**